client/results.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <h3>Home/about/etc</h3>
+    </header>
+    <section>
+        <h1>Foogle</h1>
+    </section>
+
+    <section id="oneSearchResult">
+        <div id="website"></div>
+        <div id="link"></div>
+        <div id="summary"></div>
+    </section>
```

Comment:
Nice use of id to show what this is. I would question the need to have a single`<section>` hard-coded into the HTML for the search results. Either having 10 sections hard-coded for each result, or having none and then using `document.createElement("section")` to create one for each listing when you make a call to the API.